src/components/name-tag.js

```diff
@@ -240,6 +244,32 @@ AFRAME.registerComponent("name-tag", {
     }
   },
 
+  updatePronouns() {
+    if (this.pronouns !== this.prevPronouns) {
+      this.pronounsText.el.addEventListener(
+        "text-updated",
+        () => {
+          if (this.pronounsText.el && this.nameTagText.el) {
+            this.pronounsText.el.components["text"].getSize(this.size);
```

Comment:
Here it should be `nametagText` instead of `nameTagText`. For the size evaluation I think we want to set `this.size` to the max between `nametagText`, `pronounText` and `NAMETAG_MIN_WIDTH`. We should also take the pronoun in consideration for the display name resize so maybe we can consolidate all this in `updateDisplayName` and do everything there.